### PR TITLE
Add Initial yaml for CCC 1.2.6

### PR DESCRIPTION
**Pull request by jdebacker**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jdebacker** [comment]
@conda-forge/staged-recipes This PR is ready for your review.  Thanks!

**Comment by chusloj** [comment]
@conda-forge-admin, please ping team

File: recipes/ccc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - ccc.get_taxcalc_rates
+    - ccc.styles
+    - ccc.controls_callback_script
+    - ccc.constants
+    - ccc.utils
+
+about:
+  home: https://github.com/PSLmodels/Cost-of-Capital-Calculator
+  license: CC-BY-1.0 # see SPDX identifiers
+  license_family: CC
```
Comment (dopplershift) [inline recipes/ccc/meta.yaml:45]: Probably should be `Public-Domain`

File: recipes/ccc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - ccc.calcfunctions
+    - ccc.get_taxcalc_rates
+    - ccc.styles
+    - ccc.controls_callback_script
+    - ccc.constants
+    - ccc.utils
+
+about:
+  home: https://github.com/PSLmodels/Cost-of-Capital-Calculator
+  license: CC-BY-1.0 # see SPDX identifiers
```
Comment (dopplershift) [inline recipes/ccc/meta.yaml:44]: `CC0-1.0` is what they claim in the license. See the [SPDX identifier](https://spdx.org/licenses/CC0-1.0.html).

File: recipes/ccc/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.6
+    - taxcalc >=3.1.0
+    - bokeh >=1.4.0
+    - paramtools >=0.7.1
+    - setuptools
```
Comment (dopplershift) [inline recipes/ccc/meta.yaml:26]: It looks like `setuptools` is only needed to run `setup.py`, so I *think* you can move this up under `host` instead.

File: recipes/ccc/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    - xlrd
```
Comment (dopplershift) [inline recipes/ccc/meta.yaml:27]: Do you really need to depend on this? I don't see any imports of this in the code base on GitHub, but do see a commit saying "Remove xlrd dependency"

**Review by dopplershift — CHANGES_REQUESTED** [review]

File: recipes/ccc/meta.yaml (hunk 4 above)
Comment (jdebacker) [reply recipes/ccc/meta.yaml]: Thanks for looking at this.  `xlrd` is not used in the package, but is used if one wants to recreate data contained in the package with scripts contained in repo.  Dropping this from the requirements here.

File: recipes/ccc/meta.yaml (hunk 3 above)
Comment (jdebacker) [reply recipes/ccc/meta.yaml]: Good call - moved.

File: recipes/ccc/meta.yaml (hunk 2 above)
Comment (jdebacker) [reply recipes/ccc/meta.yaml]: Updated.

File: recipes/ccc/meta.yaml (hunk 1 above)
Comment (jdebacker) [reply recipes/ccc/meta.yaml]: Updated, thanks.

**Comment by dopplershift** [comment]
Looks like cs2tc is not yet available from conda-forge. You'll need to add it, either in this PR or in another PR that gets merged first.

**Comment by jdebacker** [comment]
@dopplershift Ok - I just removed `cs2tc` as a dependency.  That's only needed when running this package in a [web application](http://compute.studio/PSLmodels/Cost-of-Capital-Calculator/) and won't need to be in the conda-fogre package as I thought it would.

**Comment by dopplershift** [comment]
Looks like the failure is because it can't find the license file. Any reason not to use the one you originally had set?

**Comment by jdebacker** [comment]
@dopplershift No - that file was fine.  I just cleaned it up and moved it to a more obvious spot in the docs in [a PR today](https://github.com/PSLmodels/Cost-of-Capital-Calculator/pull/340) and didn't realize the path to the license file for the `meta.yaml` was pointing to the location in the tarball (and not the master branch of the repo).

**Review by dopplershift — APPROVED** [review]

**Pull request merged**